### PR TITLE
最佳实践案例

### PR DESCRIPTION
**Pull request by zgq25302111**
**What this PR does**:

merge jaeger,jwt,nacos,http to dubbo samples into bestdo

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: samples/dubbogo/simple/bestdo/docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ports:
+      - 2181:2181
+    restart: on-failure
+  jaeger:
+    image: jaegertracing/all-in-one:1.26
+    ports:
+      - 5775:5775/udp
+      - 6831:6831/udp
+      - 6832:6832/udp
+      - 5778:5778
```
Comment (AlexStocks) [inline samples/dubbogo/simple/bestdo/docker/docker-compose.yml:34]: 这里为何没有保持跟上面的风格，加上协议类型？

File: samples/dubbogo/simple/bestdo/test/pixiu_test.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+package test
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
```
Comment (AlexStocks) [inline samples/dubbogo/simple/bestdo/test/pixiu_test.go:29]: 单独放进一个 import 块

File: samples/dubbogo/simple/bestdo/docker/docker-compose.yml (hunk 1 above)
Comment (zgq25302111) [reply samples/dubbogo/simple/bestdo/docker/docker-compose.yml:34]: 没写协议的默认是  http。参见 https://www.jaegertracing.io/docs/1.22/getting-started/

File: samples/dubbogo/simple/bestdo/test/pixiu_test.go (hunk 2 above)
Comment (zgq25302111) [reply samples/dubbogo/simple/bestdo/test/pixiu_test.go:29]: 单独导入了
import (
	"github.com/stretchr/testify/assert"
)

**Comment by zgq25302111** [comment]
add README.md

**Comment by mark4z** [comment]
run fmt to fix ci

**Pull request closed without merging**